src/components/POLine/POLineView.js

```diff
@@ -393,6 +394,8 @@ const POLineView = ({
           label={<FormattedMessage id="ui-orders.line.accordion.linkedAgreementLines" />}
           lineId={line.id}
         />
+
+        <LineLinkedInstances line={line} />
```

Comment:
it looks like you always render this acc, but maybe it makes sense to check some condition, to prevent back-end queries. Like, if it's a package, or something.

Reply:
queries are prevented on hook level, as for `it looks like you always render this acc` I raised this question yesterday to keep it consistent with other accordions - request won't be merged till there is a reply